.github/actions/setup/action.yml

```diff
@@ -8,6 +8,11 @@ runs:
       run: |
         sudo apt-get -yqq install libpq-dev
 
+    - name: Install image processing tools
+      shell: bash
+      run: |
+        sudo apt-get -yqq install imagemagick libvips
```

Comment:
Interesting this is not in the default setup - seems to be a change in the 24.04 runner vs the 22.04 runner